rpgame/src/main/java/pk/rpgame/view/ExplorationView.java

```diff
@@ -38,8 +38,19 @@ private int getAction() {
     return action;
   }
 
-  public void printNearestRooms(List<Room> nearestRooms) {
+  public int getRoomChoice(List<Room> nearestRooms) {
     System.out.println("Możliwe pomieszczenia, do których możesz się udać: " + nearestRooms);
```

Comment:
Jak byś miał chwilę, to możesz dodać wypisywanie całej listy pokoji jako ponumerowane menu (teraz się wyświetla jako lista w klamrach prawdopodobnie)

Reply:
też to zauważyłem, jak robiłem tłumaczenie xd